bridge/node/src/api.rs

```diff
@@ -391,6 +392,56 @@ pub fn apply_actions(mut cx: FunctionContext) -> JsResult<JsBoolean> {
 	}
 }
 
+/// Function to commit pending changes to a graph state
+/// # Arguments
+/// * `cx` - Neon FunctionContext
+/// `graph_state_id` - Unique identifier for the graph state
+/// Returns
+/// `JsResult<()>` - Empty result status
+/// Errors
+/// Does not throw
+pub fn commit(mut cx: FunctionContext) -> JsResult<JsBoolean> {
+	let graph_state_id: Handle<'_, JsNumber> = cx.argument::<JsNumber>(0)?;
+	let graph_state_id = graph_state_id.value(&mut cx) as usize;
+
+	let mut states = GRAPH_STATES.lock().unwrap();
+	let graph_state = states.get_mut(&graph_state_id);
+	if graph_state.is_none() {
+		return cx.throw_error("Graph state not found");
+	}
+	let graph_state = graph_state.unwrap();
```

Comment:
question: I've noticed we have a lot of unwraps in this file. Do these gets catches somewhere else or it might cause unexpected behavior if it happens?

Reply:
Same for all the places where unwrap is added